benchmark/CMakeLists.txt

```diff
@@ -12,5 +12,8 @@ FOREACH (F ${ALL_SRC})
     if(NOT ${prjName} STREQUAL "memtracer_performance_test")
         TARGET_LINK_LIBRARIES(${prjName} observer_static oblsm)
     endif()
+    if(${prjName} STREQUAL "memtracer_performance_test")
```

Comment:
why only put `memtracer_performance_test` in UNITTEST_OUTPUT_PATH?
you can put all benchmark binary in `benchmark` file.